trino-api/go.mod

```diff
@@ -0,0 +1,93 @@
+module trino-api
+
+go 1.21.5
+
+require (
```

Comment:
Needs cleanup.

its bloated with unnecessary libraries

for instance, it is declaring redis as a dependency, which is not used at all.